### ISSUE TITLE
Whole code jumps a bit after line 100

### ISSUE THREAD

**Issue by StefanJanssen95** [tingbot/tide-electron#18]
Because there is one extra number the line jumps when scrolling. I suggest setting a minimum width of the 3 characters (or depending on the number of lines in the file).

**Comment by joerick** [comment]
Agreed. It might be as simple as setting a CSS min-width on the gutter.
